uint16 AVOID_RED = 1301
uint16 AVOID_RED_AND_GREEN = 1302

<<<<<<< HEAD
uint16 state
=======
uint16 state
>>>>>>> 607eca33ac93d332d916be003c59f3325aee7465
